### ISSUE TITLE
Test Failure: Gradle crashes

### ISSUE THREAD

**Issue by jakelandis** [elastic/logstash#8381]
Every so often Gradle will just die with the following error. I have seen this happen on the compatibility tests which are not Docker based, and so far only on OEL and CentOS. 

```
03:10:55 FAILURE: Build failed with an exception.
03:10:55 
03:10:55 * What went wrong:
03:10:55 Gradle build daemon disappeared unexpectedly (it may have been killed or may have crashed)
```

I don't think there is anything wrong our code, but if we find this happen more often we may want to consider further examination of our build environments.

**Comment by jakelandis** [comment]
The two hosts where this was noticed have been replaced, if this continues to happen we will want to disable the daemon.

**Comment by jakelandis** [comment]
Many more cases of this today.  PR to disable it. 

Also the [Gradle doc](https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:disabling_the_daemon) advocates for disabling the daemon for CI
>At the moment, we recommend that you disable the Daemon for Continuous Integration servers as correctness is usually a priority over speed in CI environments. 

PR #8440 submitted.

**Comment by original-brownbear** [comment]
fixed via #8440
